### PR TITLE
Fix UG based on mock PE

### LINKED ISSUE #237: [PE-D] Stat function only shows Nationality
Stat function does not show gender distribution as advertised.

-------------
Labels: `type.FeatureFlaw` `severity.High`
original: NecroWolf28/ped#5

### PR DESCRIPTION
**Pull request by kevinmingtarja**
Closes #237 #227

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -454,7 +454,9 @@ Format: **`import`** `FILE_NAME.json`
 * File must be located in the `./data` directory in relative to the directory of `Socius.jar`.
 
 Examples:
-* **`import`** `amy.json` import a contact book via a file named `amy.json` into your existing application.
+* **`import`** `import.json` imports a contact book via a file named `import.json` into your existing application.
```
Comment (choonyongchan) [inline docs/UserGuide.md:457]: Actually, I have been thinking "import import.json" might not be very intuitive for first-time users, cos there is 2 'import's. Maybe we can have import groupmates.json, so it's more intuitive.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -454,7 +454,9 @@ Format: **`import`** `FILE_NAME.json`
...
+![Folder Directory for imported files](images/ImportExample.png)
```
Comment (choonyongchan) [inline docs/UserGuide.md:458]: Helpful graphics. Do you think it will be helpful to add a Windows & Linux Version of it?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -531,7 +533,7 @@ Example: `98739283`
 Example: `e/e3029834@u.nus.edu`
 
 ### `nat/NATIONALITY`
-`NATIONALITY` is only valid if it is one of the nationality listed in the [list of nationalities CSV file](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/664133/CH_Nationality_List_20171130_v1.csv/preview) by GOV.UK.
+`NATIONALITY` is only valid if it is one of the nationality listed in this [list of nationalities TXT file](https://gist.github.com/marijn/274449).
```
Comment (choonyongchan) [inline docs/UserGuide.md:536]: Perhaps we can copy paste the file into our team repo, to protect UG in the rare case this file is removed from his repo.

**Review by choonyongchan — COMMENTED** [review]
Looks good. Just some comments that we'd love to hear your thoughts about.

File: docs/UserGuide.md (hunk 1 above)
Comment (kevinmingtarja) [reply docs/UserGuide.md:457]: yeah that's a better idea actually

File: docs/UserGuide.md (hunk 2 above)
Comment (kevinmingtarja) [reply docs/UserGuide.md:458]: Yeah that would be better, perhaps anyone with windows/linux can add a screenshot for this?

File: docs/UserGuide.md (hunk 3 above)
Comment (kevinmingtarja) [reply docs/UserGuide.md:536]: Yes it's alrdy in our repo under `src/main/resources/data/nationalities.txt`

**Review by e0543978 — APPROVED** [review]
Looks good!

**Pull request merged**